### PR TITLE
Password reset

### PR DESCRIPTION
**Pull request by Alecodeco**
User password reset/recover.
Corrected some indentation and empty lines at file end.

Trello card:
https://trello.com/c/5f9dffqX

### CODE REVIEW

File: config/initializers/devise_token_auth.rb [hunk 1]
```diff
@@ -2,7 +2,12 @@
 
 DeviseTokenAuth.setup do |config|
   bypass_sign_in = false 
-  config.default_confirm_success_url = "confirmed"
+  
+  #config.default_confirm_success_url = "http://localhost:3000"
+
+  # By default, url points to testing. For production override param "redirect_url" with
+  # production webhost
+  config.default_password_reset_url = "http://localhost:3000/api/auth/users/password/edit"
```
Comment (enzofab91) [inline config/initializers/devise_token_auth.rb]: Avoid hardcoded host, if not when it is deployed in production would not work.

```suggestion
  config.default_password_reset_url = "/api/auth/users/password/edit"
```

File: config/locales/en.yml [hunk 2]
```diff
@@ -30,4 +30,5 @@
 # available at https://guides.rubyonrails.org/i18n.html.
 
 en:
-  hello: "Hello world"
+  hello: "Hello"
```
Comment (enzofab91) [inline config/locales/en.yml:33]: Remove this strings if aren't used.

File: app/controllers/api/auth/passwords_controller.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+module Api
+  module Auth
+    class PasswordsController < DeviseTokenAuth::PasswordsController
+      skip_before_action :verify_authenticity_token
+      before_action :configure_permitted_parameters, if: :devise_controller?
+
+      def edit
```
Comment (enzofab91) [inline app/controllers/api/auth/passwords_controller.rb:7]: What about a test for this? 🤔 

`GET api/v1/users/passwords/edit`

**Review by enzofab91 — COMMENTED** [review]
Good work!

Left some comments.

File: app/controllers/api/auth/passwords_controller.rb (hunk 3 above)
Comment (Alecodeco) [reply app/controllers/api/auth/passwords_controller.rb:7]: I was thinking in leaving the tests with rspec for another pull entirely !

**Pull request merged**